packages/machine-extractor/examples/tags.machine.ts

```diff
@@ -0,0 +1,13 @@
+import { createMachine } from "xstate";
+
+export const tagMachine = createMachine({
```

Comment:
are those files here like run over with the extractor and then written back and compared? in other words - is this acting as a test?

Reply:
Yes, exactly